addons/flannel/template/base/install.sh

```diff
@@ -67,6 +67,16 @@ function flannel() {
         kubectl -n kube-flannel apply -k "$dst/"
     fi
 
+    # We will remove the flannel pods to let it be re-created
+    # in order to workaround the issue scenario described in
+    # https://github.com/flannel-io/flannel/issues/1721
+    if [ "$KUBERNETES_UPGRADE" == "1" ]; then
+       sleep 30
+       log "Deleting pods from kube-flannel namespace"
+       kubectl delete --all pods --namespace=kube-flannel
+       sleep 60
```

Comment:
and here?

Reply:
After think further here. I think is safe we have this one, I added just to ensure that the bin would be created but in my tests it is very fast and take a couple of seconds. On top of that,  the `check_network` has a waiting timeout already.